### ISSUE TITLE
when using activerecord 5 schema_format = :sql still outputs to schema.rb

### ISSUE THREAD

**Issue by kjg** [thuss/standalone-migrations#129]
I just started a new project with standalone-migrations and activerecord 5. I set the schema_format = :sql, but the file created during migrations is still at schema.rb The contents of the file IS SQL however. 

Using standalone-migrations with active record 4.2 properly puts the sql in structure.sql, as does running migrations on a fresh Rails 5 install.

It appears that something about the way standalone-migrations interacts with rails 5 causes active record to output the sql to the wrong file

**Comment by kjg** [comment]
It looks like it is because of this change https://github.com/rails/rails/pull/18399/commits/99e57a8dccb700375c674c6c7872025f0022f0db in which structure dumps now use the path in `ENV['SCHEMA']` if set and standalone-migrations sets `ENV['SCHEMA']` to `db/schema.rb` by default.

**Comment by twobitlabs** [comment]
Can you test with our newest version 5.2.0? If if doesn't work for you we'd love a PR with a fix!

**Comment by kjg** [comment]
Unfortunately I no longer use standalone-migrations day to day, but I would guess a fix might involve [changing this line](https://github.com/thuss/standalone-migrations/blame/e49e3b655062d29702511565054a8c7af8122118/lib/standalone_migrations/configurator.rb#L43), and it says it hasn't been touched in 3 years.

Did you make a change that you expect to have fixed this issue? I'd be happy to take a look.

Thanks!

**Comment by twobitlabs** [comment]
No worries, I just wanted to make sure the issue didn't get left behind. I'll leave this issue open and if someone else comes along that needs that feature they can send a PR with a fix. Standalone migrations is maintained by its active users now and we just review PR's and publish updates when new PR's are merged.

**Comment by konstantinn** [comment]
Can confirm bug still exists on `~> 5.2.3`

**Comment by MaxGabriel** [comment]
You can workaround this by specifying the schema file name in the `.standalone_migrations` YAML config file:

```yaml
db:
    schema: db/schema.sql
```

You still need to specify the ActiveRecord schema format in the Rakefile:

```ruby
ActiveRecord::Base.schema_format = :sql
```

**Comment by jmehnle** [comment]
The line of code …

```ruby
      ENV['SCHEMA'] = schema
```

was originally introduced by https://github.com/thuss/standalone-migrations/pull/42. I'm not familiar enough with today's Rails/ActiveRecord code to come up with a better solution, but I figure there might be a way to hook into Rails/AR in such a way that allows us to override the path (but not the base name) of the schema/structure dump file. Then we wouldn't have to explicitly set the `SCHEMA` environment variable.

**Comment by jmehnle** [comment]
I believe PR https://github.com/thuss/standalone-migrations/pull/141 addresses this in a better way.

**Comment by twobitlabs** [comment]
Merged PR #141 and published v5.2.4. Thanks @jmehnle
